dlrover/trainer/torch/elastic_run.py

```diff
@@ -304,26 +304,46 @@ def _elastic_config_from_args(
     args,
 ) -> Tuple[ElasticLaunchConfig, Union[Callable, str], List[str]]:
     config, cmd, cmd_args = config_from_args(args)
+
+    master_config = _elastic_config_from_master(config)
     elastic_config = ElasticLaunchConfig(**config.__dict__)
 
     # PyTorch >= 2.3.0 remove log_dir in the LaunchConfig.
     if not version_less_than_230():
         elastic_config.log_dir = config.logs_specs.root_log_dir
+
     elastic_config.network_check = getattr(args, "network_check", False)
+    if master_config.network_check:
```

Comment:
No need to set 'false' if config from master is false?

Reply:
Yes. Here we assume the local command has the highest priority.